cmd2/cmd2.py

```diff
@@ -400,13 +400,13 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, persistent
                          'timing': 'Report execution times'}
 
         # Commands to exclude from the help menu and tab completion
-        self.hidden_commands = ['eof', 'eos', '_relative_load']
+        self.hidden_commands = ['eof', '_relative_load']
```

Comment:
Nice, I always felt that the **eos** command was an ugly but functional hack that I created.